Packs/Sigma/Layouts/layoutscontainer-sigma.json

```diff
@@ -368,13 +373,14 @@
                         "type": "expirationStatus",
                         "w": 1,
                         "x": 0,
-                        "y": 0
+                        "y": 0,
+                        "maxH": null
                     },
                     {
                         "displayType": "ROW",
                         "h": 2,
                         "hideName": false,
-                        "i": "default-main-Sigma Rule Layout-default-main-Sigma Rule Layout-default-main-Sigma Rule Layout-default-main-93c3e790-4293-11ef-8f1a-b9be5629a211",
+                        "i": "default-main-948860cb-b00d-463c-84a0-543e7a78107e-default-main-Sigma Rule Layout-default-main-Sigma Rule Layout-default-main-Sigma Rule Layout-default-main-93c3e790-4293-11ef-8f1a-b9be5629a211",
```

Comment:
What are all of these changes?

Reply:
![image](https://github.com/user-attachments/assets/04a8cd88-55dd-4007-958f-ac5439b50e0b)